### PR TITLE
Implemented error log

### LINKED ISSUE #120: Error messages
Many of our error messages are too long, and don't fit in the notification window. We need some way of displaying or capturing these errors in full. They are important for debugging purposes.

### PR DESCRIPTION
**Pull request by Mingling94**
Also reduced down some commenting on private variables in daemonManager

Fixes #120

### CODE REVIEW

File: js/uiManager.js [hunk 1]
```diff
@@ -93,7 +86,7 @@ function UIManager() {
 			// eTooltip.hide();
 			eTooltip.animate({
 				'opacity':'0'
-			},400,function() {
+			}, 400,function() {
```
Comment (DavidVorick) [inline js/uiManager.js]: weird to have a space after the first comma, but not the second

Comment (Mingling94) [inline js/uiManager.js]: Oops

File: js/uiManager.js [hunk 2]
```diff
@@ -197,6 +190,14 @@ function UIManager() {
 	 * clicking the notification
 	 */
 	this.notify = function notify(message, type, clickAction) {
+		// Record errors for reference
+		if (type === 'error') {
+			if (!errorLog) {
+				errorLog = Fs.createWriteStream(Path.join(__dirname, 'errors.log'));
```
Comment (lukechampine) [inline js/uiManager.js:196]: does this append properly? We don't want to overwrite previous errors

Comment (Mingling94) [inline js/uiManager.js:196]: It does append properly.

**Comment by Mingling94** [comment]
boop @DavidVorick @lukechampine

**Pull request merged**